android/build.gradle

```diff
@@ -60,6 +60,6 @@ android {
 
 dependencies {
     implementation "org.jetbrains.kotlin:kotlin-stdlib-jdk7:$kotlin_version"
-    implementation "com.getphyllo:phyllo-connect:0.3.1"
-    //implementation "com.getphyllo:phyllo-connect:0.3.2.17-SNAPSHOT"   
+    //implementation "com.getphyllo:phyllo-connect:0.3.1"
+    implementation "com.getphyllo:phyllo-connect:0.3.2.1-SNAPSHOT"   
```

Comment:
Update to a release version instead of snapshot version

Reply:
we are going to testing so that final PR will be raise from release_v0.3.* to main , at time we will have change in release version , its not yet release so